recipes/xtb-python/meta.yaml

```diff
@@ -16,7 +16,6 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
```

Comment:
this might still fail: https://github.com/conda-forge/cf-mark-broken/pull/67

Reply:
you can try to set version 7.3.0 here.

Reply:
Looks like it was successful with GCC 7.5.0

Reply:
cool. resolving this part of the PR comments.